port/unix/omrsysinfo.c

```diff
@@ -3380,6 +3384,26 @@ retrieveLinuxMemoryStatsFromProcFS(struct OMRPortLibrary *portLibrary, struct J9
 		} /* end if else-if */
 	} /* end while() */
 
+	swappinessFs = fopen(PROC_SYS_VM_SWAPPINESS, "r");
+	if (NULL == swappinessFs) {
+		Trc_PRT_retrieveLinuxMemoryStats_failedOpeningSwappinessFs(errno);
+		rc = OMRPORT_ERROR_SYSINFO_ERROR_SWAPPINESS_OPEN_FAILED;
+		goto _cleanup;
+	}
+
+	rc_swappiness = fscanf(swappinessFs, "%" SCNu64, &memInfo->swappiness);
+	if (1 != rc_swappiness) {
+		if (EOF == rc_swappiness) {
+			Trc_PRT_retrieveLinuxMemoryStats_failedReadingSwappiness(errno);
+		} else {
+			Trc_PRT_retrieveLinuxMemoryStats_unexpectedSwappinessFormat(1, rc);
+		}
+		rc = OMRPORT_ERROR_SYSINFO_ERROR_READING_SWAPPINESS;
+		goto _cleanup;
+	} else {
+		rc = 0; /* Set it to 0 to indicates success */
+	}
```

Comment:
- the `else` block is no longer needed
- `rc_swappiness` -> `rcSwappiness`
- `Trc_PRT_retrieveLinuxMemoryStats_unexpectedSwappinessFormat` should take `rcSwappiness` instead of `rc`?
```suggestion
	rcSwappiness = fscanf(swappinessFs, "%" SCNu64, &memInfo->swappiness);
	if (1 != rcSwappiness) {
		if (EOF == rcSwappiness) {
			Trc_PRT_retrieveLinuxMemoryStats_failedReadingSwappiness(errno);
		} else {
			Trc_PRT_retrieveLinuxMemoryStats_unexpectedSwappinessFormat(1, rcSwappiness);
		}
		rc = OMRPORT_ERROR_SYSINFO_ERROR_READING_SWAPPINESS;
		goto _cleanup;
	}
```